tools/xabuild/XABuild.cs

```diff
@@ -37,7 +37,7 @@ static int Main ()
 							var symbolicLink = symbolicLinks [i];
 							var systemDirectory = systemDirectories [i];
 							Console.WriteLine ($"[xabuild] creating symbolic link '{symbolicLink}' -> '{systemDirectory}'");
-							if (!SymbolicLink.Create (symbolicLink, systemDirectory)) {
+							if (!SymbolicLink.Create (systemDirectory, symbolicLink)) {
```

Comment:
I think this actually broke the build.

I think the parameters seem backwards, because the Win32 API also seems backwards: https://github.com/xamarin/xamarin-android/blob/32007a1d72df700b267ce61955f3ba60d9f2166d/tools/xabuild/SymbolicLink.cs#L55

I would like to eventually completely remove these symbolic links from xabuild, Ankit got some changes in MSBuild that are in 15.8 where we can do away with them.

Reply:
Or maybe it's unix that seems backwards? https://github.com/xamarin/xamarin-android/blob/32007a1d72df700b267ce61955f3ba60d9f2166d/tools/xabuild/SymbolicLink.cs#L58

I don't know, it's Friday, it definitely works or our builds wouldn't be green at all.